doc/source/tutorial/stats/rv_infrastructure.md

```diff
@@ -0,0 +1,772 @@
+    m_, v_ = data.mean(), data.var()
+    scale = np.sqrt(v_ / v)
+    loc = m_ - scale*m
+    return loc, scale
+
+fit_loc_scale(X, data)
+```
+
+Note that the estimates are quite poor in this example, and poor performance of the heuristic factored into the decision not to provide a replacement.
+
+The last method of the old infrastructure is `fit`, which estimates the location, scale, and shape parameters of an underlying distribution given a sample from the distribution.
+
+```{code-cell} ipython3
+params = stats.weibull_min.fit(data)
+params
+```
+
+The convenience of this method is a blessing a curse. When it works, the simplicity is much appreciated. For some distributions, analytical expressions for the [maximum likelihood estimates](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation) (MLE) of the parameters have been programmed manually, and for these distributions, the `fit` method is quite reliable. For most distributions, however, fitting is performed by numerical minimization of the negative log-likelihood function. This is not guaranteed to be successful - both because of inherent limitations of MLE and the state of the art of numerical optimization - and in these cases, the user of `fit` is stuck. However, a modicum of understanding goes a long way toward ensuring success, so here we present a mini-tutorial of fitting using the new infrastructure. 
+
+First, note that MLE is one of several potential strategies for fitting distributions to data. It does nothing more than find the values of the distribution parameters for which the negative log-likelihood (NLLF) is minimized. Note that for the original data, the NLLF is:
+
+```{code-cell} ipython3
+stats.weibull_min.nnlf((c, loc, scale), data)
+```
+
+The NLLF of the parameters estimated using `fit` are lower:
+
+```{code-cell} ipython3
+stats.weibull_min.nnlf(params, data)
+```
+
+Therefore, `fit` considers its job done, whether or not this satisfies the user's notions of a "good fit" or whether the parameters are within reasonable bounds.
+
+At its simplest, then, it is not so hard to replicate what `fit` does using tools available in SciPy. As discussed above, the NLLF is given by:
+
+```{code-cell} ipython3
+def nllf(params):
+    c, loc, scale = params
+    X = Weibull(c=c) * scale + loc
+    return -X.logpdf(data).sum()
+
+nllf(params)
+```
+
+To perform the minimization, we can use `scipy.optimize.minimize`.
+
+```{code-cell} ipython3
+from scipy import optimize
+eps = 1e-10  # numerical tolerance to avoid invalid parameters
+lb = [eps, -10, eps]  # lower bound on `c`, `location`, and `scale`
+ub = [10, np.min(data)-eps, 10]  # upper bound on `c`, `location`, and `scale`
+x0 = [1, 1, 1]  # guess to get optimization started
+bounds = optimize.Bounds(lb, ub)
+res = optimize.minimize(nllf, x0, bounds=bounds)
+res
+```
+
+Although the value of `fun` (the NLLF) is not quite as low as with the solution provided by `weibull_min.fit`, the PDFs are not so different.
```

Comment:
From the top:

> I plan to add quite a bit more about fitting.

Maybe if I haden't done made all the improvements to the existing `fit` this would have been an easier sell. Alas.

Reply:
I made some improvements. I would like to expand with examples of fitting with other objective functions and entirely different requirements (e.g. users have asked to fit a distribution not to *data* but such that specific statistics assume particular values), but I think it is much more convincing. I didn't even consciously choose this case to be problematic for MLE, but it happens to be a great example of the sort of pitfals I've seen in "bug" reports.